### PR TITLE
More friendly and clear error message

### PR DESCRIPTION
**Pull request by canyie**
Just change the word a bit to make people happier 😃 
#6022

### CODE REVIEW

**Comment by osm0sis** [comment]
Maybe "restore back to stock boot image and try again" or something?

File: scripts/boot_patch.sh [hunk 1]
```diff
@@ -185,8 +185,8 @@ rm -f ramdisk.cpio.orig config magisk*.xz
 for dt in dtb kernel_dtb extra; do
   if [ -f $dt ]; then
     if ! ./magiskboot dtb $dt test; then
-      ui_print "! Unsupported boot image $dt"
-      abort "! Please restore back to stock boot image"
+      ui_print "! Boot image $dt was patched by old (unsupported) Magisk"
+      abort "! Please restore back to stock boot image and try again"
```
Comment (yujincheng08) [inline scripts/boot_patch.sh:189]: How about
```suggestion
      abort "! Please manually patch the *stock* boot image and flash it"
```

File: scripts/boot_patch.sh [hunk 2]
```diff
@@ -185,8 +185,8 @@ rm -f ramdisk.cpio.orig config magisk*.xz
 for dt in dtb kernel_dtb extra; do
   if [ -f $dt ]; then
     if ! ./magiskboot dtb $dt test; then
-      ui_print "! Boot image was patched by old (unsupported) Magisk"
-      abort "! Please restore back to stock boot image"
+      ui_print "! Boot image $dt was patched by old (unsupported) Magisk"
+      abort "! Please restore back to stock boot image and try again"
```
Comment (canyie) [inline scripts/boot_patch.sh:189]:
```suggestion
      abort "! Please try again with *stock* boot image"
```

Maybe better?

Comment (osm0sis) [reply scripts/boot_patch.sh:189]: To be honest stock might confuse people on custom ROMs. How about "unpatched" ?

Comment (canyie) [reply scripts/boot_patch.sh:189]: Sounds good!

```suggestion
      abort "! Please try again with *unpatched* boot image"
```

**Review by osm0sis — APPROVED** [review]

**Comment by olivluca** [comment]
> And maybe that next line could also be more clear with "Please restore back to stock boot image and try again" or something?

Both this new message and issue #6022 left me wondering what I had to do.
What I did (and it seems it worked) was to tap the uninstall button in the magisk app, select the option to "restore image", disregard the dire warning that all modules would be uninstalled (it didn't happen) then tap the install button.
Simple once you know how, difficult if guided by the error message.

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**